libtracepoint-control-cpp/include/tracepoint/TracepointSession.h

```diff
@@ -1252,13 +1252,14 @@ namespace tracepoint_control
             tracepoint_decode::PerfEventMetadata const& metadata,
             TracepointEnableState enableState) noexcept(false);
 
+        static uint32_t
+        CalculateBufferCount(TracepointSessionOptions const& options) noexcept;
+
         static std::unique_ptr<BufferInfo[]>
         MakeBufferInfos(
             uint32_t bufferCount,
             uint32_t pageSize,
-            _In_reads_(cpuBufferSizesCount) uint32_t const* cpuBufferSizes,
-            uint32_t cpuBufferSizesCount,
-            uint32_t perCpuBufferSize) noexcept(false);
+            TracepointSessionOptions const& options) noexcept(false);
```

Comment:
Note: All of the old parameters are fields in options, so just pass options instead.